fe_rtos/Cargo.toml

```diff
@@ -21,5 +20,8 @@ features = ["spin_no_std"]
 version = "0.1.2"
 features = ["alloc"]
 
+[target.'cfg(target_arch = "arm")'.dependencies]
+cortex-m = "0.6"
```

Comment:
Is this as specific as we can get? If we ever intend on building for ARM platforms other than cortex-m, this would still get pulled in as a dependency/

Reply:
As far as I can tell it is. According to https://doc.rust-lang.org/1.29.1/std/arch/index.html the only options for arm in target_arch are arm and aarch64.
If we need to in the future, we can add features for the different 32-bit arm architectures

Reply:
related: https://github.com/rust-lang/rust/issues/44036#issuecomment-324062733